XTB: Support interest and fractional shares

gabrielgradinaru opened the pull request:
## Added

Add support in XTB for interest and fractional shares

dickwolff reviewed (commented):
Can you bump the GitVersion with a patch number?

gabrielgradinaru commented:
Sorry, I totally forgot about unit tests.

I've fixed the xtb test and also updated yahoo-finance2 package because it was failing due to https://github.com/gadicc/node-yahoo-finance2/issues/764

dickwolff commented:
LGTM!
